libavcodec/qsvenc.c

```diff
@@ -1355,6 +1429,12 @@ static int qsv_retrieve_enc_params(AVCodecContext *avctx, QSVEncContext *q)
         q->exthevctiles_idx = ext_buf_num;
         ext_buffers[ext_buf_num++] = (mfxExtBuffer*)&hevc_tile_buf;
     }
+#if QSV_HAVE_HE
+    if (q->dual_gfx) {
+        q->exthypermodeparam_idx = ext_buf_num;
+        ext_buffers[ext_buf_num++] = (mfxExtBuffer*)&hyper_mode_param_buf;
+    }
```

Comment:
MediaSDK runtime doesn't support mfxExtHyperModeParam.  In order to avoid error when using MediaSDK runtime,  please add a check for runtime version.

Reply:
Slightly modified the PR. Everything protected by QSV_ONEVPL define.
https://github.com/intel-media-ci/ffmpeg/blob/9bbd3b3b17487a37a4c9669e17ec3ac5968da978/libavcodec/qsv_internal.h#L68

Reply:
Added runtime version check